src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -238,6 +238,57 @@ Similarly, the OPTIONAL [`rec-<label>`](../99-appendices/09-entities.md#rec)
+| True transverse relaxation time map          | T2map     | In seconds (s). T2 maps are REQUIRED to use this suffix regardless of the method used to generate them.                                                                                                                                                                                                                                                                                                                                                                                                           |
+| Observed transverse relaxation time map      | T2starmap | In seconds (s). T2-star maps are REQUIRED to use this suffix regardless of the method used to generate them.                                                                                                                                                                                                                                                                                                                                                                                                      |
+| Longitudinal relaxation rate map             | R1map     | In seconds<sup>-1</sup> (1/s). R1 maps are REQUIRED to use this suffix regardless of the method used to generate them.                                                                                                                                                                                                                                                                                                                                                                                            |
+| True transverse relaxation rate map          | R2map     | In seconds<sup>-1</sup> (1/s). R2 maps are REQUIRED to use this suffix regardless of the method used to generate them.                                                                                                                                                                                                                                                                                                                                                                                            |
+| Observed transverse relaxation rate map      | R2starmap | In seconds<sup>-1</sup> (1/s). R2-star maps are REQUIRED to use this suffix regardless of the method used to generate them.                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Is there a benefit to supporting images that are direct transformations of one another? If so, then I think it may be worth mentioning the relationship between T2starmap and R2starmap, R2map and T2map, and R1map and T1map, in the descriptions for each pair. Also... what about reordering the rows so that matching pairs are placed together?

Reply:
Yes, depending on the research interest, reporting rate would be more favourable than reporting time. I reordered the table and briefly expressed how `R`s are related to `T`s 7fa7e07.

Reply:
Sounds good. Thanks!